pixiedust/display/chart/renderers/bokeh/histogramDisplay.py

```diff
@@ -54,20 +54,24 @@ def getChartOptions(self):
         ]
 
     def canRenderChart(self):
-        valueFields = self.getValueFields()
-        if len(valueFields) != 1:
-            return (False, "You can only select one value")
-        else:
-            return (True, None)
+        return (True, None)
 
     def getExtraFields(self):
         color = self.options.get("color")
         return [color] if color is not None else []
     
     def createBokehChart(self):     
-        
         binsize = int(self.options.get('binsize', 10))  
-        return Histogram(self.getWorkingPandasDataFrame(), values=self.getValueFields()[0],
-            plot_width=800,bins=binsize, color=self.options.get("color"), 
-            xgrid=True, ygrid=True
-        )
\ No newline at end of file
+        valueFields = self.getValueFields()
+        histograms = []
+        if len(valueFields) != 1:
+            for i in valueFields:
+                histograms.append(Histogram(self.getWorkingPandasDataFrame(), values=i,
+            plot_width=400, plot_height=400,bins=binsize, color=self.options.get("color"), 
```

Comment:
Instead of hard-coding the width and height, can you please use self.getPreferredOutputWidth() and self.getPreferredOutputHeight() which return the total width of the output area based on the client resolution